README.md

```diff
@@ -4,177 +4,8 @@
 
 The Universal Application Tool (UAT) aims to simplify the application process for benefits programs by re-using applicant data for multiple benefits applications. It is being developed by Google.org in partnership with the City of Seattle.
 
-* [Technology overview](#technology-overview)
```

Comment:
suggestion (non-blocking): Could we include the TOC in the Wiki? I found it really helpful

Reply:
done!